pkg/agent/openflow/network_policy.go

```diff
@@ -1451,12 +1454,14 @@ func parseMetricFlow(flow string) (uint32, types.RuleMetric) {
 
 func (c *client) NetworkPolicyMetrics() map[uint32]*types.RuleMetric {
 	result := map[uint32]*types.RuleMetric{}
-	ovsctlClient := ovsctl.NewClient(c.nodeConfig.OVSBridge)
-	egressFlows, _ := ovsctlClient.DumpTableFlows(uint8(EgressMetricTable))
-	ingressFlows, _ := ovsctlClient.DumpTableFlows(uint8(IngressMetricTable))
+	egressFlows, _ := c.ovsctlClient.DumpTableFlows(uint8(EgressMetricTable))
+	ingressFlows, _ := c.ovsctlClient.DumpTableFlows(uint8(IngressMetricTable))
+	// There could be other flows like default flow and traceflow flows in the table.
+	// Only metrics flows are supposed to have normal priority.
+	metricFlowIdentifier := fmt.Sprintf("priority=%d,", priorityNormal)
```

Comment:
How about defining a global variable to save some cost for every call?

Reply:
done